### ISSUE TITLE
Raid Skip Info

### ISSUE THREAD

**Issue by iXerxes** [SavedInstances/SavedInstances#766]
I'm not sure if this feature fits SavedInstances very well, but it would be incredibly useful if it could display information about raid skip progress across characters.

Currently, I use a small WeakAura that I made to check for it, but this requires me to login to each character to check the info.

![image](https://i.imgur.com/GPpKJM8.png)

![image](https://i.imgur.com/YMknQRw.png)

This isn't a complete list, as I made it a while ago. There are other skips for Shadowlands and Dragonflight.
Each skip has a quest associated with it, and the API functions used are:
- GetQuestObjectiveInfo
- C_QuestLog.IsQuestFlaggedCompleted

If you'd consider this feature, I'd be happy to find the quest ID's for all eligible raids for you :)
Thanks!

**Comment by iXerxes** [comment]
I figured out how your custom quests are written, so I modified your presets, and this is what I came up with:

![image](https://i.imgur.com/E5G2nnM.png)

![image](https://i.imgur.com/eUZAcrR.png)

![image](https://i.imgur.com/mXkQHTK.png)

![image](https://i.imgur.com/fHw1o50.png)

![image](https://i.imgur.com/DnLzOn9.png)

The way it's displayed covers everything I could think of. It's a little messy, I know, but I don't know how else to optimise the display.

I understand this wouldn't really fit into SavedInstances by default, since it's very personal, but maybe there could be some kind of toggle for optional extras within the addon? It could toggle the display of additional presets such as this one. 

I'd be happy to give you the code if this is something you'd consider adding. Thanks :)

**Comment by LiangYuxuan** [comment]
I love the idea but the display might need to be adjusted to reduce line width. There're some complaint about it.

I can confirm that most of skip can be unlocked by upper difficulty.

**Comment by Alfakennyone** [comment]
Hey @iXerxes could you pass that code? I just requested the idea and the creator linked to this 


https://github.com/SavedInstances/SavedInstances/issues/825

**Comment by iXerxes** [comment]
Hi, @LiangYuxuan. I agree, the line width is an issue. I'm not so sure what adjustments to make though :/

> I can confirm that most of skip can be unlocked by upper difficulty.

Yes, this is true, and my code makes use of this feature. If you complete Heroic, both Normal and Heroic will display a checkmark. Completing Mythic will replace everything with a checkmark, regardless of lesser progress

Before I do, is it ok for me to post modified versions of your files here? Just want to check with you first, before I share my raid skip code :)

**Comment by LiangYuxuan** [comment]
> Before I do, is it ok for me to post modified versions of your files here? 

Yes, feel free to do so.

I think we can put most of the progress in hover tooltip, and the main tooltip only shows the best difficulty's progress or complete mark.

**Comment by Alfakennyone** [comment]
Yeah, putting it in the Raid hover tooltip would be nice - similar to the Emissary Quest

![image](https://github.com/SavedInstances/SavedInstances/assets/61605997/b83e4ebc-c7a9-4daf-b5e8-12bd8fe9452d)

Have it look something like this as I previously shown

The fewer the addons the better lol so this would be a nice touch imo

**Comment by Alfakennyone** [comment]
Yeah that's why I'm hoping @LiangYuxuan can implement it more permanently

Would be very useful but thanks for the code @iXerxes

**Comment by iXerxes** [comment]
As much as I like this addition, I don't think it belongs in SavedInstances. Not by default, anyway. Perhand some setting within the menu to enable/disable extra features, such as this one.

**Comment by Alfakennyone** [comment]
> As much as I like this addition, I don't think it belongs in SavedInstances by default. Perhaps some setting within the menu to enable/disable extra features, such as this one.

I think it does as it pertains to tracking instance information


Anyways, can you update the code in your repositories? :) thank you

**Comment by iXerxes** [comment]
@Alfakennyone https://github.com/iXerxes/SI-RaidSkips/tree/live

**Comment by Alfakennyone** [comment]
> This is why I think it would be better suited to some sort of "optional features" setting within SavedInstances, where it can be enabled/disabled.

Yeah, like the settings you provided. That works
 
> The updated version: https://github.com/iXerxes/SI-RaidSkips/tree/live

Thank you!

I just noticed, it's missing the Nyalotha skip lol

**Comment by Alfakennyone** [comment]
Well, raid skip quests are account wide now so no need for this =p
